environment.yml

```
@@ -29,6 +29,7 @@ dependencies:
   - redis-py==5.0.1
   - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
   - scrapy==2.11.1
+  - h5netcdf==1.3.0
```

```suggestion
  - botocore>=1.33.10 # Minimum version that is compatible with python >= 3.10 is botocore>=1.13.0
  - h5netcdf==1.3.0
```
Could you please remove scrapy in this PR, since it's in newzealidar now